datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/types/container/mappers/ContainerMapper.java

```diff
@@ -39,6 +40,8 @@ public static Container map(final EntityResponse entityResponse) {
     final Container result = new Container();
     final Urn entityUrn = entityResponse.getUrn();
     final EnvelopedAspectMap aspects = entityResponse.getAspects();
+    Long lastIngested = SystemMetadataUtils.getLastIngested(aspects);
```

Comment:
One thing to keep in mind -- There is a possibility that in the future we become smarter about which aspects we fetch here. At that time we'll need to make some decisions...

Reply:
totally - always good to keep that in mind!